tests/config/pytest_modules.yml

```diff
@@ -2522,6 +2522,14 @@ ngscheckmate/vafncm:
   - modules/nf-core/ngscheckmate/vafncm/**
   - tests/modules/nf-core/ngscheckmate/vafncm/**
 
+nonpareil/curve:
+  - modules/nf-core/nonpareil/curve/**
+  - tests/modules/nf-core/nonpareil/curve/**
+
+nonpareil/nonpareil:
+  - modules/nf-core/nonpareil/nonpareil/**
+  - tests/modules/nf-core/nonpareil/nonpareil/**
```

Comment:
No need to test in both pytest and nf-test

Reply:
I was told to keep them in for now? It lints with an error if they aren't included I think?

Reply:
OK I was wrong!